app/src/main/java/org/feedhenry/welcome/MainActivity.java

```diff
@@ -204,166 +204,150 @@
  */
 package org.feedhenry.welcome;
 
-import android.os.Bundle;
-import android.support.design.widget.NavigationView;
-import android.support.design.widget.NavigationView.OnNavigationItemSelectedListener;
-import android.support.v4.app.Fragment;
-import android.support.v4.app.FragmentManager;
-import android.support.v4.view.GravityCompat;
-import android.support.v4.widget.DrawerLayout;
-import android.support.v7.app.ActionBar;
-import android.support.v7.app.AppCompatActivity;
+import android.os.*;
```

Comment:
Is there a memory footprint side-effect from this change to import the whole library and not just the needed components from the library?

Reply:
I think that it depends on settings in Android Studio codestyle. It's usually set that from some count of imports, it will be converted to wildcard notation.